src/commands/runScripts/_app-server.js

```diff
@@ -23,20 +24,18 @@ const runServer = require(`${process.cwd()}/scripts/app-server`); // TODO: move
 const { argv } = yargs.implies('cold-start', 'host');
 
 const getServerAppMap = () => {
-	return Object.keys(argv)
-		.filter(a => locales.includes(a))
-		.reduce((map, localeCode) => {
-			const importPath = argv[localeCode];
-			map[localeCode] = require(importPath).default;
-			return map;
-		}, {});
+	const renderer = require(path.resolve(
```

Comment:
doesn't this remove the ability to pass in locales through arg's?

Reply:
Which makes sense if we are going for just combined or english only... but want to validate the intent

Reply:
in dev, yes - all languages will be available automatically
> it avoids having to hassle with running the app with `--locales` options